app/javascript/api/listingsApiService.ts

```diff
@@ -30,7 +30,7 @@ export const getEligibilityQueryString = (
 ) => {
   const getIncomeLevel = () => {
     return filters?.income_timeframe === "per_month"
-      ? filters?.income_total * 12
+      ? filters.income_total * 12
```

Comment:
Removed the redundant question mark, code coverage analysis was identifying an untested branch of code.